content/MBC5.md

```diff
@@ -47,3 +47,20 @@ maps the corresponding external RAM bank (if any) into the memory area at
 A000-BFFF.
 
 ![](imgs/MBC5.png "imgs/MBC5.png")
+
+### Rumble
+
+On cartridges which feature a rumble motor, bit 3 of the RAM Bank register
+is connected to the Rumble circuitry instead of the RAM chip. Setting the
+bit to 1 enables the rumble motor and keeps it enabled until the bit is reset again.
+
+To control the rumble's intensity, it should be turned on and off repeatedly,
```

Comment:
Is this some kind of PWM technique? Do we know more details on how fast the switch could happen (duty-cycle)?

Reply:
The rumble motor rotates quite intensely when turned on, so the frequent on-and-offs are just needed to not make it shake too much. The Pokémon Pinball examples show the "Intense" Rumble Setting flipping between on and off approximately every 17.5ms, so pretty close to a simple on/off switch per frame, which seems to work pretty well on hardware.